doc/markdownlinkconverter/README.md

```diff
@@ -28,11 +28,26 @@ happens in 2 passes, which is needed because there can be files with no title.
 
 ## Link Validation
 
+### internal links
+
 The link validation works with a simple try to `fopen` the file,
 which the link refers to.
 
+### external links
+
+Every link starting with `http`, `https` or `ftp` will be written to a file named `httpLinks.txt` located in your
+build folder. With the following syntax:
+
+	<file>|<line> col 0 | <url>
+
+Note: Due to the nature of the Markdown Link Converter the file can only be opened in append mode. So delete it and rerun the
+html build process (`make clean` could be needed) to get a list without duplicates.
```

Comment:
What do you mean? You cannot create an empty file after removing it? Actually the file mode "a" should create a file if the file does not exist.